src/main/java/de/tum/in/www1/exerciseapp/repository/ResultRepository.java

```diff
@@ -31,6 +31,9 @@
     @Query("select r from Result r where r.completionDate = (select min(rr.completionDate) from Result rr where rr.participation.exercise.id = r.participation.exercise.id and rr.participation.student.id = r.participation.student.id and rr.successful = true) and r.participation.exercise.course.id = :courseId and r.successful = true order by r.completionDate asc")
     List<Result> findEarliestSuccessfulResultsForCourse(@Param("courseId") Long courseId);
 
+    // Hibernate returns List<Object[]> when selecting multiple columns
```

Comment:
please also provide a more detailed comment using JavaDoc notation, explain the sql statement and the values in the object array